cdap-kubernetes/pom.xml

```diff
@@ -112,6 +112,12 @@
       <artifactId>cdap-proto</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
```

Comment:
We should avoid having k8s extension depending on cdap-common which pulls in a lot of dependencies and is fairly large.

Reply:
I've removed the dependency. The reason for adding this dependency was to share property names which need to be set by KubeMasterEnvironment and read by the Main Service that I added. Now I've maintained 2 copies of the property names, one in each file. Can't figure out where to define these constants to make them sharable. If I add them as static strings in KubeMasterEnv, I'll need to add cdap-kubernetes dependency in cdap-master which isn't desirable. Can you suggest a better way than duplicating the strings in both files?

Reply:
Passing kay-value properties with specific prefix defined in MasterEnvironmentContext interface. MainService fetches all properties with the prefix and passes them to JMX Metrics Collector. This way no additional dependencies are required. cdap-master-spi has the prefix constant.